services/event/models/event.go

```diff
@@ -12,7 +12,8 @@ type Event struct {
 }
 
 type EventLocation struct {
-	Description string  `json:"description" validate:"required"`
-	Latitude    float64 `json:"latitude"    validate:"required"`
-	Longitude   float64 `json:"longitude"   validate:"required"`
+	Description string   `json:"description" validate:"required"`
+	Tags        []string `json:"tags"        validate:"required,dive,oneof=SIEBEL0 SIEBEL1 SIEBEL2 SIEBEL3 SIEBEL4 ECEB1 ECEB2 ECEB3 ECEB4 ECEB5 KENNEY DCL"`
```

Comment:
As an FYI. This starts to break down the API as a generic solution for events. RP, for example, may need a different set of tags. I'd recommend reading this in from a config file.

Reply:
Alternatively, we could just keep it simple and remove the `oneof=...` from the validation entirely